packages/gazelle_cli/lib/commons/functions/confirmation.dart

```diff
@@ -0,0 +1,17 @@
+import 'dart:io';
+
+/// Asks for confirmation before deleting a project.
+class Confirmation {
+  /// Asks for confirmation before deleting a project.
+  ///
+  /// Returns `true` if user confirms, `false` otherwise.
+
+  bool getConfirmation(String message) {
+    print('$message (y/N)');
+    String? answer = stdin.readLineSync();
+    if (answer?.toLowerCase() == 'y' || answer?.toLowerCase() == 'yes') {
+      return true;
+    }
+    return false;
+  }
+}
```

Comment:
I would remove the class and just have a top-level function
called `bool getConfirmation(String message)`.
Just like the other functions in that directory.

Reply:
The class is deleted and now getConfirmation is now the top-level function